### PR TITLE
fix(MovableCoord): change bind/unbind logic

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#231 

## Details
- change bind/unbind logic.
- previous logic could not apply dynamic options

## Preferred reviewers
@netil and others

### CODE REVIEW

File: test/unit/js/movableCoord.test.js [hunk 1]
```diff
@@ -1233,4 +1231,90 @@ test("_convertInputType (not support touch)", function() {
...
+	// Then
+	equal(this.inst.isActivate(), false, "check deactivation");
+	deepEqual(beforePos, this.inst._pos, "check whether position is moved");
+
+	// When
+	this.inst.setBy(10,10);
+
+	// Then
+	deepEqual(beforePos, this.inst._pos, "check whether position is moved");
+
```
Comment (jongmoon) [inline test/unit/js/movableCoord.test.js]: It will be more good to add `setTo` check after activate()

**Comment by jongmoon** [comment]
LGTM. 

And Later..., It could be better to test frequent activate/deactivate on Low performance device (by hammer create/destroy issues) although I think it will not make problem.

**Comment by sculove** [comment]
@jongmoon I add testcases. therefore, I could find a bug. so, I fix some code. thanks

File: test/unit/js/movableCoord.test.js [hunk 2]
```diff
@@ -1284,7 +1284,7 @@ test("check deactivation", function() {
 });
 
 
-test("after deactivation, call methods", function() {
+test("after deactivation, call setTo/By methods", function() {
```
Comment (jongmoon) [inline test/unit/js/movableCoord.test.js]: after deactivation --> after activation

File: test/unit/js/movableCoord.test.js [hunk 3]
```diff
@@ -1284,7 +1284,7 @@ test("check deactivation", function() {
...
 	//Given
 	var before = this.inst.isActivate();
```
Comment (jongmoon) [inline test/unit/js/movableCoord.test.js]: above code is not used.

**Comment by jongmoon** [comment]
LGTM

**Pull request merged**